source/ui/BioXAS/BioXASSidePersistentView.cpp

```diff
@@ -61,7 +72,7 @@ BioXASSidePersistentView::~BioXASSidePersistentView()
 void BioXASSidePersistentView::onCalibrateEnergyButtonClicked()
 {
 	bool inputOK = false;
-	double newEnergy = QInputDialog::getDouble(this, "Monochromator Energy Calibration", "Enter current calibrated energy:", BioXASSideBeamline::bioXAS()->mono()->energyControl()->value(), 0, 10000000, 1, &inputOK, Qt::Sheet);
+	double newEnergy = QInputDialog::getDouble(this, "Monochromator Energy Calibration", "Enter current calibrated energy:", BioXASSideBeamline::bioXAS()->mono()->energyControl()->value(), -10000000, 10000000, 1, &inputOK, Qt::Sheet);
```

Comment:
The value itself was arbitrarily chosen. At present, it's possible to have a negative energy reading on the beamline, which is why the minimum here is negative.